src/code/z_actor.c

```diff
@@ -1544,6 +1544,7 @@ u32 Actor_HasParent(Actor* actor, GlobalContext* globalCtx) {
     }
 }
 
+// GI_NONE is used for picking things up, GI_MAX for catching in a bottle.
 s32 func_8002F434(Actor* actor, GlobalContext* globalCtx, s32 getItemId, f32 xzRange, f32 yRange) {
```

Comment:
Imo this isn't really the right place to put this comment, because it's mostly a hacky way the function was used rather than what it's intended for, especially regarding GI_MAX since any value >= GI_MAX would have the exact same effect.

Instead I suggest including a mention above the calls that use special values like these, to 'justify' their usage.

Reply:
This comment was intended to be descriptive rather than prescriptive. Looking again, I think the reverse is true, actually: GI_MAX is used consistently for this by the 4 relevant actors. But GI_NONE isn't used consistently: EnKusa doesn't use it, for example, but EnIshi, EnNiw, and ObjTsubo do.

Reply:
I agree with roman here, a comment above the call explaining the hacky usage of the GI value seems more appropriate than putting it above the function itself

Reply:
Okay, how would you like these comments phrased?
```
// This is used for catching the actor in a bottle
func_8002F434(&this->actor, globalCtx, GI_MAX, 80.0f, 20.0f);
```
and
```
// This is used for lifting the actor
func_8002F434(&this->actor, globalCtx, GI_NONE, 80.0f, 20.0f);
```
?

Reply:
`GI_MAX` in this case allows the player to catch the actor in a bottle
`GI_NONE` in this case allows the player to lift the actor

just trying to put emphasis on the gi values being unusual and not used for their actual purpose